src/CLI/CLI.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <AssemblyName>AxeWindowsScanner</AssemblyName>
+    <RootNamespace>AxeWindowsScanner</RootNamespace>
+  </PropertyGroup>
+
+  <Import Project="..\..\build\NetStandardRelease.targets" />
+
+  <ItemGroup>
+    <PackageReference Include="CommandLineParser" Version="2.7.82" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Automation\Automation.csproj" />
+  </ItemGroup>
+
+  <Target Name="WriteAxeWindowsVersionFile">
```

Comment:
Why did this need to be added here? This should have been written already by the first project to be compiled.

Reply:
Copy/Paste error. Now removed.